### ISSUE TITLE
Get Everpad into the Debian repository

### ISSUE THREAD

**Issue by e2jk** [nvbn/everpad#387]
Would allow all Debian derivatives (including Ubuntu) to get it straight out of their repos, without having to go through a PPA.

This issue is meant as a master issue, to list all the steps needed to get Everpad into the official repository.
- ~~#388 Provide a manpage~~
- #382 Depend on python-thrift instead of providing it.
- #255 Package must be cross-architecture
- #136 Separating the several components into different packages (the master `everpad` package would only install Ubuntu-specific stuff on Ubuntu)
- to be continued

**Comment by e2jk** [comment]
I have created an Intent To Package bug report on the Debian bug tracker: [#739637](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=739637)

**Comment by philsf** [comment]
Kudos on the effort, @e2jk! Any recent progress in this package?

**Comment by e2jk** [comment]
No, I've stopped using Evernote/Everpad mid 2014 (see [comment 36 on the Debian bug](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=739637#36)). I'm not aware of anyone else working on this recently, and the Debian bug had not received any attention since my last message.

**Comment by philsf** [comment]
How far did you go on that package? Did you manage it to work, for any snapshot of upstream?

I did a naive attempt to compile your package, but it added two or three wrong deps (e.g. `python-py-oauth2` instead of `python-oauth2`), as well as the correct deps. It shouldn't be hard to fix, provided I manage to find where the  error comes from.

It would be much easier if I knew, say, a specific commit where the package fully works, and then work my way forward. ;-)

**Comment by e2jk** [comment]
I don't really remember how far. You should just start over again, picking
stuff out where possible and check how the upstream Ubuntu package is set
up.

Good luck.
Emilien
